src/components/timeline/TimelineMarkerCanvas.js

```diff
@@ -179,11 +173,16 @@ class TimelineMarkerCanvas extends PureComponent {
               }
             }
           } else {
+            const radiusRatio = w / h;
+            // TODO Should we use an exponential scale instead ?
```

Comment:
I'd say try it real quick and see if it makes more sense. I'd rather choose one rather than leaving a dangling TODO in the code.